src/client/components/Swipe/SwipeCard/SwipeCard.jsx

```diff
@@ -9,7 +9,7 @@ const SwipeCard = props => {
     <div className="swipe-card" style={{ zIndex }}>
       <img className="card-image" src={place.image_link} />
       <div className="card-info">
-        <InfoIcon className="info-icon" onClick={showModal}/>
+        <InfoIcon className="info-icon" onClick={() => showModal(place.place_id)}/>
```

Comment:
i'm not a fan of having a function to more than one thing. ideally it should follow the **Single Responsibility Principle.**

right now `showModa()` does two things:
-  getting the place info
-  changing state of `isModalShown`

one suggestion is you can create a separate handler function that does what you need on click
```js
onClick{this.handleClick(place.place_id)}
```

```js
handleClick() {
  const { showModal, getPlaceData } = this.props;
   showModal();
   getPlaceData(place.place_id); // you'd have to pass in the props from parent component
}